### ISSUE TITLE
AWSPowerShell module (v4.1.121) cannot be installed - authenticode signature of the file 'AWSPowerShell.psd1' is not valid

### ISSUE THREAD

**Issue by jack-cloud-platform** [aws/aws-tools-for-powershell#277]
### Describe the bug

The latest version of the AWSPowerShell PowerShell module (4.1.121) cannot be installed, an error of `The module 'AWSPowerShell' cannot be installed or updated because the authenticode signature of the file 'AWSPowerShell.psd1' is not valid.` is returned.

### Expected Behavior

The AWSPowerShell PowerShell module (version 4.1.121) should be able to be installed without errors of invalid authenticode signatures.

### Current Behavior

AWSPowerShell PowerShell module cannot be installed when running:
Install-Module AWSPowerShell -RequiredVersion 4.1.121

### Reproduction Steps

PowerShell ran:
```
Install-Module AWSPowerShell -RequiredVersion 4.1.121
Y (Are you sure you want to install the modules from
'PSGallery'?)
```

Error returned:
```
PackageManagement\Install-Package : The module 'AWSPowerShell' cannot be installed or updated because the authenticode
signature of the file 'AWSPowerShell.psd1' is not valid.
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:1809 char:21
+ ...          $null = PackageManagement\Install-Package @PSBoundParameters
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package],
   Exception
    + FullyQualifiedErrorId : InvalidAuthenticodeSignature,ValidateAndGet-AuthenticodeSignature,Microsoft.PowerShell.P
   ackageManagement.Cmdlets.InstallPackage
```

### AWS Tools for PowerShell version used

AWSPowerShell 4.1.121

### PowerShell version used

Name                           Value
----                           -----
PSVersion                      5.1.19041.1682
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1682
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

### Operating System and version

Windows 10

**Comment by henrivdr01** [comment]
We got the same error. We had to force the previous version with -MaximumVersion 4.1.120

**Comment by towerbe** [comment]
Same issue when trying to install from the PowerShell Gallery.

**Comment by ashovlin** [comment]
Thank you for the report, we're investigating. 

[Version 4.1.121 of AWSPowerShell.NetCore](https://www.powershellgallery.com/packages/AWSPowerShell.NetCore/4.1.121) does have a valid signature if you need to stay on the latest version and are able to switch modules.

4.1.121 of `AWS.Tools.Common` is impacted also, which therefore impacts all of the `AWS.Tools.<service>` modules.

**Comment by ashovlin** [comment]
We just published version 4.1.122 to PowerShell Gallery, which was signed with a root certificate which is trusted on Windows.
